File: eimui/src/main/java/com/xjl/eimui/entry/AudioRecordWaterWaveEntry.java
package com.xjl.eimui.entry;

public class AudioRecordWaterWaveEntry {

    public String chat_up_move_cancel;
    public String chat_up_cancel;

    public AudioRecordWaterWaveEntry(){
        chat_up_move_cancel="Slide up to cancel";
        chat_up_cancel="chat_up_cancel";
    }

    public AudioRecordWaterWaveEntry(String chat_up_move_cancel, String chat_up_cancel) {
        this.chat_up_move_cancel = chat_up_move_cancel;
        this.chat_up_cancel = chat_up_cancel;
    }
}

File: eimui/src/main/java/com/xjl/eimui/entry/AudioRecordWaterWaveEntry.kt
package com.xjl.eimui.entry

import android.content.Context
import com.xjl.eimui.R

class AudioRecordWaterWaveEntry {

    var chatUpMoveCancel: String
    var chatUpCancel: String

    constructor(context: Context) {
        chatUpMoveCancel = context.resources.getString(R.string.chat_up_move_cancel)
        chatUpCancel = context.resources.getString(R.string.chat_up_cancel)
    }

    constructor(chatUpMoveCancel: String, chatUpCancel: String) {
        this.chatUpMoveCancel = chatUpMoveCancel
        this.chatUpCancel = chatUpCancel
    }
}
